config/metasrv.example.toml

```diff
@@ -2,3 +2,4 @@ bind_addr = '127.0.0.1:3002'
 server_addr = '127.0.0.1:3002'
 store_addr = '127.0.0.1:2379'
 datanode_lease_secs = 15
+selector_type = 'LeaseBased'
```

Comment:
use comment to show all available options like

## selector: LeaseBased, LoadBased